.github/workflows/validate.yaml

```diff
@@ -112,7 +112,7 @@ jobs:
                 -level="error"
 
       - name: Go Lint - knative-operator
-        uses: golangci/golangci-lint-action@v2
+        uses: golangci/golangci-lint-action@v3
```

Comment:
>v3.0.0+ requires explicit setup-go installation step prior to using this action: uses: actions/setup-go@v2.
The skip-go-installation option has been removed.

Fixes the issue with too fresh go version being installed for golangci-lint runs.